File: JS Essentials/3.0 Conditional Statements/07. Area of Figures.js
function areaOfFigures(arg) {
    let [figure, numA, numB] = [...arg.map(el => isNaN(el) ? el : Number(el))]
    let result = 0
  
    switch (figure) {
        case 'square': result = Math.pow(numA, 2); break 
        case 'rectangle': result = numA * numB ; break
        case 'circle': result = numA * numA * Math.PI; break
        case 'triangle': result = numA * numB / 2; break
    }
          
    return  result.toFixed(3)
}
  
// console.log(areaOfFigures(['square', 5]))

File: Python Essentials/3.0 Conditional Statements/07. Area of Figures.py
def area_of_figures(figure):
    num_a = float(input())
    num_b = 0
    if figure == 'rectangle' or figure == 'triangle':
        num_b = float(input())

    switch = {
        'square': num_a * num_a,
        'rectangle': num_a * num_b,
        'circle':  num_a * num_a * 3.14159,
        'triangle': num_a * num_b / 2,
    }
          
    return  f'{switch[figure]:.3f}'


print(area_of_figures(input()))

